### ISSUE TITLE
Support passing a separate value for the thumb

### ISSUE THREAD

**Issue by Maxr1998** [2307vivek/Seeker#36]
It'd be useful to be able to pass the value for the thumb separately, so that it can have its own progress while seeking.
See this screenshot from the YouTube app for an example use case:
![Screenshot](https://github.com/2307vivek/Seeker/assets/5903699/38506949-17ba-4b32-9407-bcd75d676a9c)

If necessary, I can look into implementing this myself.

**Comment by Maxr1998** [comment]
Yes, this should be possible by adding a `thumbValue` to the Seeker function, and setting the default of the parameter to `value`. That way, it'll stay optional.

**Comment by 2307vivek** [comment]
And then the `thumbValue` will be updated in the `onValueChange` lambda. Sounds good.

You want to do this yourself?

**Comment by Maxr1998** [comment]
Yes, I'd be open to do that.

**Comment by 2307vivek** [comment]
Ok, I am assigning this to you.
